### ISSUE TITLE
Glider Socks5 UDP not working with client mode using username and password

### ISSUE THREAD

**Issue by TGSAN** [nadoo/glider#219]
Glider Socks5 UDP not working with client mode using username and password
  
Do some tests:  
| Test | Socks5 client on 1080 port test result |
| --- | --- |
| [glider --listen=socks5://user:pass@:1080] | TCP Pass, UDP Pass |
| [glider --listen=socks5://user:pass@:1090] - [glider --listen=socks5://:1080 --forward=socks5://user:pass@127.0.0.1:1090] | TCP Pass, UDP Failed |

**Comment by nadoo** [comment]
Only not working on windows?

**Comment by TGSAN** [comment]
Sorry, there is a little problem with my previous description.
I edited this issue, this problem occurs on any system (Windows and Linux).
After retesting, I found that this problem only occurs in glider client mode with socks5 username and password.

run `glider --listen=socks5://user:pass@:1090 --verbose`
and run `/glider --listen=socks5://:1080 --forward=socks5://user:pass@127.0.0.1:1090 --verbose`
use socks5 client to connect the socks5://user:pass@127.0.0.1:1090, both TCP and UDP will work.
but use socks5 client to connect the socks5://127.0.0.1:1080, UDP is not work.
